lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
```

Comment:
> Though, I think some additional comments in here may be beneficial for explaining how to use the class, kinda like in the docs I wrote:

> https://github.com/rails-api/active_model_serializers/pull/1241/files#diff-584dd3531b6814be034e8a6087f82fe9R22

> then, our real docs could just link to this file for an example.
> Or maybe we could link to this file and some tests which show some basic usage of this base model class.

I think a combination of the above is good. Maybe I can leave this more or less as is, then in your pr:  add docs that reference the rubydoc link (which will then be live), a link to the source, and add usage comments to the source.